CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 11.1.3
+
+### Fixed
+
+- #1976 Fix MountProvider fetching paths on oracle @CarlSchwan
```

Comment:
Not a blocker but this is having a bit of a rendering issue on the App Store pages due to https://github.com/Python-Markdown/markdown/issues/848

Reply:
Moved to the end and rebased to include #1988